setup.py

```diff
@@ -15,15 +15,16 @@
 from setuptools import setup
 
 setup(
-    name='gprof2dot',
-    version='2015.02.03',
+    name='yelp-gprof2dot',
+    version='1.0.0',
     author='Jose Fonseca',
```

Comment:
Sure. I didn't see this pattern repeated on some of our other forks I looked at, like [hacheck](https://github.com/Yelp/sqlparse/blob/master/setup.py) and [kafka-python](https://github.com/Yelp/kafka-python/blob/master/setup.py), so I left it the way it is. As long as I leave the copyright alone it should be fine, though.